tests/core/tools/array.js

```diff
@@ -151,8 +151,35 @@
 				assert.areSame( 0, index );
 				arrayAssert.itemsAreSame( [ 1234 ], array );
 			} );
-		}
+		},
+
+		'test array.find': function() {
```

Comment:
Technically you're doing two test cases in one. Please, split it into two clearly defined test cases.